### PR TITLE
feat: add 2024-01-01 sdk

### PR DESCRIPTION
**Pull request by RenzoPrettoMS**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/chaos/Azure.ResourceManager.Chaos/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (live1206) [inline sdk/chaos/Azure.ResourceManager.Chaos/package-lock.json:1]: This file is not needed

File: sdk/chaos/Azure.ResourceManager.Chaos/src/autorest.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ azure-arm: true
 csharp: true
 library-name: Chaos
 namespace: Azure.ResourceManager.Chaos
-#tag: package-2023-11
-require: https://github.com/Azure/azure-rest-api-specs/blob/4af52aaac2c3b4af4a0e61378d33c5bc050e65e2/specification/chaos/resource-manager/readme.md
+tag: package-2024-01
```
Comment (live1206) [inline sdk/chaos/Azure.ResourceManager.Chaos/src/autorest.md:10]: The specified tag is same as the default tag in readme.md, should be removed or commented out.

Comment (RenzoPrettoMS) [reply sdk/chaos/Azure.ResourceManager.Chaos/src/autorest.md]: Good catch, pushed up change now!

Comment (RenzoPrettoMS) [reply sdk/chaos/Azure.ResourceManager.Chaos/src/autorest.md]: Thank you @live1206

File: sdk/chaos/Azure.ResourceManager.Chaos/package-lock.json (hunk 1 above)
Comment (RenzoPrettoMS) [reply sdk/chaos/Azure.ResourceManager.Chaos/package-lock.json]: Removed

**Comment by RenzoPrettoMS** [comment]
It seems like the build started failing after I made the requested changes, do you know why this could happen?

**Review by live1206 — APPROVED** [review]

**Pull request merged**